pycaption/scc/__init__.py

```diff
@@ -327,6 +327,9 @@ def _handle_double_command(self, word):
             if word == self.last_command:
                 self.last_command = ''
                 return True
+            elif _is_pac_command(word) and _is_pac_command(self.last_command):
```

Comment:
This logic was used to ignore the second duplicate command, not the first as it seems to be your intention (I saw you wrote a test called test_skip_first_pac_command).

You go with the first command, it doesn't find a previous double so it returns False (as in the command is NOT a double) and it gets processed.
Then the second command comes, it is compared to the first one, it is identified as a double, return True (the command IS a double) and doesn't get processed.

I realise now that a better name for this would have been _is_double_command for it to be easier to understand